deploy/charts/kmesh-helm/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: kmesh
+name: kmesh-helm
```

Comment:
why do you change it

Reply:
Since there is already an image named kmesh in the registry, you should use a new name for the helm package.
You could porposal with a better name.

Reply:
Ah, then if user install kmesh using the oci, it seems not convenient. how does envoygateway do

Reply:
see https://hub.docker.com/r/envoyproxy/gateway-helm/tags

Reply:
the installation command will looks like:
```
helm install kmesh oci://ghcr.io/kmesh-net/kmesh-helm --version v0.0.0-latest -n kmesh-system --create-namespace
```

Reply:
The name of Kmesh daemonset will be the same as this name, which causes the e2e test to fail.

We should decouple the two.

Reply:
`fullnameOverride` should help.